pages/faq/troubleshooting/faq.md

```diff
@@ -78,6 +77,10 @@ Additionally, you should whitelist the following domains for the relevant ports
 * `*.docker.com`
 * `*.docker.io`
 
+##### Can I use {{ $names.cloud.lower }} in China?
```

Comment:
```suggestion
##### Can I use {{ $names.cloud.lower }} in countries with restrictive national firewalls?
```

Reply:
might be better to avoid naming names, just a thought!

Reply:
That was my thought too, which is why I only included that bit in a FAQ section, where it probably is appropriate but I'll let someone else decide :)

Reply:
I suspect some people will search for China specifically; that's the country we tend to get asked about. Maybe "countries with restrictive firewalls such as China?"